Makefile

```diff
@@ -0,0 +1,8 @@
+bootstrap:
+	.buildscript/bootstrap.sh
+
+# use yarn scripts for the rest
+%:
```

Comment:
This just means you now prefix commands with "make" rather than "npm run". I'm wondering why Segment feels this was necessary, since everything seemed to be "OK" before hand.

Reply:
The idea here is to have a standard interface for working between libraries. Typically we want `make bootstrap`, `make dependencies`, `make test` to work out of the box (e.g. we have the same targets in the Go library https://github.com/segmentio/analytics-go/blob/v3.0/Makefile).


This is to make it easier for supports agents don't have to hunt down how to run the install step (which can look like `pod install`, `mvn package`, `yarn`, etc.).


As to the implementation, I figured that rather than declare something like this:
```
test:
  yarn run standard && yarn run nyc ava
```

it would be better to re-use the scripts defined in https://github.com/segmentio/analytics-node/blob/master/package.json#L27. And not duplicate ourselves/get out of sync. This also has the advantage that more familiar node users can run `yarn test`, while less experience users with Node may run `make test.